### PR TITLE
Little Red Riding Hooded Mercenary

### PR DESCRIPTION
**Pull request by monochromatism**
## About The Pull Request

Adds Little Red Riding Hooded Mercenary. You know who she is.

She comes with the following:
- Powerful... power generation... abilities.
- Solid EGO. Very fun to use.
- EXTREMELY interactive in playable abnormalities, including player UI cues for what the AI-controlled abnormality has as automatic behavior.
- Unique dynamic with Reddened Buddy and Blue Smocked Shepherd. (May be buggy, this was *exceptionally* messy to code.)

Please keep in mind that all her specific numbers (damage values, HP thresholds, cooldowns, etc.) are subject to change; it's hard to get a concept of exactly how strong she is, aside from that she can consistently beat Blue Smocked Shepherd and Reddened Buddy in combat. They're all modular and prepared for quick changes, however.

## Why It's Good For The Game

See above. More abnormalities. Completing a contribution to Abno Jam of May 2023. Adds something unique to the current dynamic. Shouldn't be too hard to code Wolf interactivity, if and when Big and Will Be Bad Wolf arrives.

## Changelog
:cl:
add: Little Red Riding Hooded Mercenary
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/he/blue_shepherd.dm [hunk 1]
```diff
@@ -259,6 +260,13 @@
 		if(L == awakened_buddy && !buddy_hit)
 			buddy_hit = TRUE //sometimes buddy get hit twice so we check if it got hit in this slash
 			awakened_buddy.adjustHealth(700) //it would take approximatively 9 slashes to take buddy down
+		if(istype(L, /mob/living/simple_animal/hostile/abnormality/red_hood))
+			if(!red_hit)
+				red_hit = TRUE
+				var/mob/living/simple_animal/hostile/abnormality/red_hood/current_red = L
+				current_red.WatchIt()
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/he/blue_shepherd.dm:267]: I'd have an issue with this if it was a WAW vs an HE, honestly.

**Review by dsmith328 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/abnormality/waw/red_riding_mercenary.dm [hunk 2]
```diff
@@ -0,0 +1,686 @@
...
+	rapid_melee = 2
+	speed = 0.5
+	threat_level = WAW_LEVEL
+	work_chances = list( // Red's counter is unaffected by work, so her rates matter less than usual. I've upped them from base LC to encourage working her more.
+		ABNORMALITY_WORK_INSTINCT = list(50, 55, 60, 65, 70),
+		ABNORMALITY_WORK_INSIGHT = list(40, 45, 50, 55, 60),
+		ABNORMALITY_WORK_ATTACHMENT = 45,
+		ABNORMALITY_WORK_REPRESSION = 45,
+		"Request" = 100
+		)
```
[the remark this answers is not in the document]
Comment (dsmith328) [reply code/modules/mob/living/simple_animal/abnormality/waw/red_riding_mercenary.dm:27]: TLDR; She's swol and doesn't need these boosts. Her utility pays for it in droves.

**Comment by LanceSmites328** [comment]
This PR has had changes requested to it for **4 Months** and nothing has been done. If this has not been addressed in the next 3 days I'll be closing this PR and giving someone else a chance to take over this abnormality.

**Comment by LanceSmites328** [comment]
Marking as stale and closing PR.

**Pull request closed without merging**